server/main_server/src/main.cpp

```diff
@@ -1056,10 +1073,12 @@ Environment Variables:
         }
         catch (const irods::exception& e) {
             log_server::error("{}: Reload error: {}", __func__, e.client_display_what());
+            irods::notify_service_manager("READY=1\nSTATUS=Reload error");
             return;
         }
         catch (const std::exception& e) {
             log_server::error("{}: Reload error: {}", __func__, e.what());
+            irods::notify_service_manager("READY=1\nSTATUS=Reload error");
```

Comment:
just noticing that these two are 'Reload error'... where the ones above are 'Reload failed'...

Reply:
Yeah, at this point in the reload operation, the reload itself is technically done (or at least that's what it looks like to me) but the post-reload tasks such as killing off the old agent factory can still fail.